src/App.js

```diff
@@ -1,25 +1,63 @@
-import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json';
+import { useState } from 'react';
+import FoodBox from './components/FoodBox'
+import AddFoodForm from './components/AddFoodForm'
+import SearchForm from './components/SearchForm'
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+  
+  const [viewedFoods, setViewedFoods] = useState(foods)
+
+  const addFoodEntry = (foodEntry) => {
+    setViewedFoods((prevList) => {
+      const newList = [...prevList];
+      newList.push(foodEntry)
+      return newList;
+    })
+  }
+
+  const searchFood = ((searchWord) => {
```

Comment:
about how updating list of food after deleting letters from search bar, you already know how to manage it after you asked in class 
you can also reset the form inputs to empty string after adding a new food :) for having better UI